one-node/molecule.yml

```diff
@@ -0,0 +1,19 @@
+---
+dependency:
+  name: galaxy
+  requirements_file: ../requirements.yml
+
+driver:
+  name: docker
+
+docker:
+  containers:
+  - name: omero-all
```

Comment:
[Travis CI](https://travis-ci.org/ome/ansible-examples-omero/jobs/458473781) is failing with

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/bin/molecule", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/cli.py", line 41, in main
    cli(obj={})
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/test.py", line 96, in test
    util.sysexit(t.execute()[0])
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/test.py", line 44, in execute
    status, output = c.execute(exit=False)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/create.py", line 42, in execute
    self.molecule.driver.up(no_provision=True)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/driver/dockerdriver.py", line 133, in up
    self._build_ansible_compatible_image(container)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/driver/dockerdriver.py", line 260, in _build_ansible_compatible_image
    'image of {}:{} ...').format(container['image'],
KeyError: 'image'
```

Which base image should be used here?

Reply:
Uh, I assume this is a change in molecule, no?